### PR TITLE
Portfolio: Finish mobile version.

### PR DESCRIPTION
**Pull request by maximoortelli**
_Portfolio: Portfolio: finish mobile version is a new project where I will finish my first portfolio. The goal here is to master all the tools and best practices learned in the previous steps to use in all Microverse projects and probably my future work as well._

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,6 +122,541 @@ <h4>LET'S CONNECT</h4>
...
+
+            </div>
+
+            <div id="imagen"><img src="media/escritoriotrabajo4.png" alt="" class="escritorio-trabajo-foto-izquierda"></div>
+         
+         </div>
+
+
+     </div>
+
```
Comment (usorfaitheloho) [inline index.html]: Kindly implement these changes for the works section of your project :

-  The container of the section should be a section tag to show good HTML semantics😄

File: index.html [hunk 2]
```diff
@@ -122,6 +122,541 @@ <h4>LET'S CONNECT</h4>
...
+                    
+                     <img src="media/sigue.png" alt="flecha" id= "sigue">
+
+                 </li>
+
+             </ul>
+
+         </div>
+
+     </div>
```
Comment (usorfaitheloho) [inline index.html]: Kindly ensure you implement these changes :

- Please ensure the container of this section is a `section` tag for a good HTML semantics 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @maximoortelli  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI design ✔️
> * Followed HTML and CSS semantics ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (maximoortelli) [reply index.html]: Done👍

File: index.html (hunk 2 above)
Comment (maximoortelli) [reply index.html]: Done👍

**Review by Munsa1 — APPROVED** [review]
Hi @maximoortelli, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes From Previous Reviewer :heavy_check_mark: 
 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**